insights/specs/default.py

```diff
@@ -622,6 +622,7 @@ def lsmod_all_names(broker):
     mysqld_limits = foreach_collect(mysqld_pid, "/proc/%s/limits")
     named_checkconf_p = simple_command("/usr/sbin/named-checkconf -p")
     namespace = simple_command("/bin/ls /var/run/netns")
+    ndctl_list = simple_command("/usr/bin/ndctl list -Ni")
```

Comment:
Per the manual of `ndctl` there are various different `ndctl list` sub-commands, would you please rename this spec more specifically?